themes/OpenFisca-2022/layouts/_default/showcase.html

```diff
@@ -6,8 +6,24 @@
         <div class="grid grid-hasPaddingBottom">
             <div class="cards">
             {{ range $.Site.Data.showcase }}
-                {{ $description := index .description $.Site.Language.Lang }}
-                {{ partial "card.html" (dict "title" .title "description" $description "country" .country "website" .website "author" .author "Site" $.Site) }}
+                {{ if not .until }}
+                    {{ $description := index .description $.Site.Language.Lang }}
+                    {{ partial "card.html" (dict "title" .title "description" $description "country" .country "website" .website "author" .author "Site" $.Site) }}
+                {{ end }}
+            {{ end }}
+            </div>
+        </div>
+    </div>
+
+    <div class="wrapper">
+        <div class="grid grid-hasPaddingBottom">
+            <h2 class="grid_col-12">{{ .Params.disused_reuses }}</h2>
+            <div class="cards mt-l">
+            {{ range sort $.Site.Data.showcase ".until" }}
+                {{ if .until }}
```

Comment:
These two lines seem to contain a redundant filtering logic.

Reply:
As seen earlier together, thes sort is needed to classify obsolete reuses by date.

However, it would be interesting to remove the `if .until` condition by doing a `range where`, but I haven't managed to make it work yet.

Reply:
Ok i finaly found why we can't do `range where $.Site.Data.showcase "until" "!=" nil`:  `range` accepts maps, arrays and slices, `first` and `where` only accept an array or a slice. Clearly explain in [this post](https://discourse.gohugo.io/t/ranging-over-collection-of-data-files/14488/9).